eva_submission/nextflow/variant_load.nf

```diff
@@ -119,3 +120,69 @@ process load_vcf {
     java -Xmx4G -jar $params.jar.eva_pipeline --spring.config.location=file:$params.eva_pipeline_props --parameters.path=$variant_load_properties
     """
 }
+
+
+vcf_files_list = Channel.fromPath(params.valid_vcfs)
+            .splitCsv(header:true)
+            .map{row -> tuple(file(row.vcf_file), row.db_name)}
+
+
+/*
+ * Create properties files for Accession Import Job.
+ */
+process create_properties_for_acc_import_job {
+    input:
+    tuple vcf_file, db_name from vcf_files_list
```

Comment:
I think this process won't wait for the variant load to be completed before being executed. 
You need to create a dependency between them

Reply:
added dependency